plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -185,6 +205,11 @@ def api_check_param(command, name):
     return name in api.Command[command].params
 
 
+def api_check_version(requested_version):
```

Comment:
Would it make sense to be able to use something like `api_check_version(4.8, "<")`?

Reply:
It is not obvious at the moment that `api_check_version` will do a `>=` comparison.

Reply:
I changed this function to accept a string with the operator to be used to compare versions. Now, it is more flexible and clear on what is being done.

I also changed the function name to api_check_ipa_version, as the IPA version is what is being evaluated, not the IPA API version.